### PR TITLE
feat: [PLG-2997]: updating intitials for names for companies

### PR DESCRIPTION
**Pull request by dhruvUpadhyay07**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

File: src/modules/10-common/components/ResourceCenter/utils.ts [hunk 1]
```diff
@@ -38,6 +38,18 @@ export const openZendeskSupport = (e: React.MouseEvent<Element, MouseEvent>): vo
   window.open(HARNESS_SUPPORT_LINK)
 }
 
+const initializeList = ['disney']
+
+const getInitials = name => {
```
Comment (kapil-harness) [inline src/modules/10-common/components/ResourceCenter/utils.ts:43]: I think for username 'Kapil.choudhary@harness.io' the output will be 'kci'

@dhruvUpadhyay07 lets first split on the basis of '@' then split on the basis of '.' to get the initials of first and last name

**Review by kapil-harness — CHANGES_REQUESTED** [review]

File: src/modules/10-common/components/ResourceCenter/utils.ts (hunk 1 above)
Comment (dhruvUpadhyay07) [reply src/modules/10-common/components/ResourceCenter/utils.ts:43]: We are doing that if you check the canny object

Comment (kapil-harness) [reply src/modules/10-common/components/ResourceCenter/utils.ts:43]: ok, makes sense

File: src/modules/10-common/components/ResourceCenter/utils.ts [hunk 2]
```diff
@@ -50,7 +62,11 @@ const initCanny = (currentUserInfo: UserInfo, account?: AccountDTO) => {
       ],
       email: currentUserInfo.email,
       id: currentUserInfo.uuid,
-      name: currentUserInfo.name
+      name: initializeList.includes(account?.companyName.toLowerCase())
```
Comment (kapil-harness) [inline src/modules/10-common/components/ResourceCenter/utils.ts:65]: This change is not universal, so where is the list of accounts for which you have to do this change?

**Review by kapil-harness — CHANGES_REQUESTED** [review]

File: src/modules/10-common/components/ResourceCenter/utils.ts (hunk 2 above)
Comment (dhruvUpadhyay07) [reply src/modules/10-common/components/ResourceCenter/utils.ts:65]: Its declared at the top

**Review by kapil-harness — APPROVED** [review]

File: src/modules/10-common/components/ResourceCenter/utils.ts [hunk 3]
```diff
@@ -38,6 +38,18 @@ export const openZendeskSupport = (e: React.MouseEvent<Element, MouseEvent>): vo
   window.open(HARNESS_SUPPORT_LINK)
 }
 
+const initializeList = ['disney']
```
Comment (kapil-harness) [inline src/modules/10-common/components/ResourceCenter/utils.ts:41]: @dhruvUpadhyay07 are we sure that company name for Disney is 'disney'? don't you think accountId would be a better check?

**Review by kapil-harness — CHANGES_REQUESTED** [review]

**Comment by bot2-harness** [comment]
This PR has been automatically closed due to inactivity.

**Pull request closed without merging**